modules/collections/client/controllers/collections.controllers.js

```diff
@@ -22,6 +22,14 @@ collectionModules.controller('CollectionEditCtrl',
       $scope.project = project;
       $scope.types = types;
 
+      // ensure sorting for display purposes
```

Comment:
This is necessary because the database retrieval doesn't seem to return the documents already sorted.